products/monitoring/api.yaml

```diff
@@ -964,6 +962,141 @@ objects:
+        - rolling_period
+        - calendar_period
+      description: |
+        A calendar period, semantically "since the start of the current
+        <calendarPeriod>".
+      values:
+        - DAY
+        - WEEK
+        - FORTNIGHT
+        - MONTH
+    - !ruby/object:Api::Type::NestedObject
+      name: serviceLevelIndicator
+      description: |
+        serviceLevelIndicator (SLI) describes a good service.
+        It is used to measure and calculate the quality of the Service's
+        performance with respect to a single aspect of service quality.
+      properties:
+        - !ruby/object:Api::Type::NestedObject
+          name: basicSli
+          exactly_one_of:
+            - basic_sli
+          description: |
+            Basic Service-Level Indicator (SLI) on a well-known service type.
+            Performance will be computed on the basis of pre-defined metrics.
+
+            SLIs are used to measure and calculate the quality of the Service's
+            performance with respect to a single aspect of service quality.
+          properties:
+            - !ruby/object:Api::Type::Array
+              name: method
+              description: |
+                An optional set of RPCs to which this SLI is relevant.
+                Telemetry from other methods will not be used to calculate
+                performance for this SLI. If omitted, this SLI applies to all
+                the Service's methods. For service types that don't support
+                breaking down by method, setting this field will result in an
+                error.
+              item_type: Api::Type::String
+            - !ruby/object:Api::Type::Array
+              name: location
+              description: |
+                An optional set of locations to which this SLI is relevant.
+                Telemetry from other locations will not be used to calculate
+                performance for this SLI. If omitted, this SLI applies to all
+                locations in which the Service has activity. For service types
+                that don't support breaking down by location, setting this
+                field will result in an error.
+              item_type: Api::Type::String
+            - !ruby/object:Api::Type::Array
+              name: version
+              description: |
+                 The set of API versions to which this SLI is relevant.
+                 Telemetry from other API versions will not be used to
+                 calculate performance for this SLI. If omitted,
+                 this SLI applies to all API versions. For service types
+                 that don't support breaking down by version, setting this
+                 field will result in an error.
+              item_type: Api::Type::String
+            - !ruby/object:Api::Type::NestedObject
+              name: latency
+              description: |
+                Parameters for a latency threshold SLI.
+              exactly_one_of:
```

Comment:
same

Reply:
this one I'm happier to change but eventually the basicSli will contain a non-empty availability block (see https://cloud.google.com/monitoring/api/ref_v3/rest/v3/services.serviceLevelObjectives#BasicSli).